src/lifecycles/bootstrap.js

```diff
@@ -3,16 +3,24 @@ import {
   BOOTSTRAPPING,
   NOT_MOUNTED,
   SKIP_BECAUSE_BROKEN,
+  toName,
 } from "../applications/app.helpers.js";
 import { reasonableTime } from "../applications/timeouts.js";
 import { handleAppError, transformErr } from "../applications/app-errors.js";
+import { addProfileEntry } from "../devtools/profiler.js";
 
 export function toBootstrapPromise(appOrParcel, hardFail) {
+  let startTime;
+
   return Promise.resolve().then(() => {
     if (appOrParcel.status !== NOT_BOOTSTRAPPED) {
       return appOrParcel;
     }
 
+    if (__PROFILE__) {
+      startTime = Date.now();
```

Comment:
Do you want to consider using `performance.now()` instead? https://developer.mozilla.org/en-US/docs/Web/API/Performance/now

Reply:
Nice, I didn't know about performance.now(). The one potential downside is you lose absolute timestamps (2021-09-14 00:00:00) in favor of timestamps that are relative to the [time origin](https://developer.mozilla.org/en-US/docs/Web/API/DOMHighResTimeStamp#the_time_origin). But that might even be better, it's not clear that it's really a downside. So I'll switch them over.